### PR TITLE
Update to latest Specter Desktop

### PR DESCRIPTION
**Pull request by AaronDewes**

### CODE REVIEW

File: apps/specter-desktop/docker-compose.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ x-logging:
 
 services:
   web:
-    image: lncm/specter-desktop:v1.2.2@sha256:22d6abe4ba97dcb9d3f5f1695524c5262be176d873f5f57c9d9c3a95840cb3da
+    image: lncm/specter-desktop:v1.3.1@sha256:c72fc1f270d6e5f8426f185e5c08fb8581728ad5c6ec943d719a9d08ce996c15
```
Comment (willcosgrove) [inline apps/specter-desktop/docker-compose.yml:5]: Can I ask where you got the sha256 digest for this? I was trying to open a PR for this myself, and I couldn't locate where the one for the previous version came from, and so I couldn't be sure I was getting the correct one for the updated version. And I am not able to locate this specific digest you have listed here.

I'm looking at this page: https://hub.docker.com/layers/lncm/specter-desktop/v1.3.1/images/sha256-c24068af20f6a3fde96be9128827fde1e638db32c7bfde0e0f53e1be063caffb

None of the architectures on that page have the SHA you posted, which makes me think I'm looking in the wrong place.

**Comment by willcosgrove** [comment]
I would love to see this get merged in soon. There's a bugfix in 1.3.1 that resolves an issue causing wallet pages to crash the browser when there are a lot of transactions.

File: apps/specter-desktop/docker-compose.yml (hunk 1 above)
Comment (AaronDewes) [reply apps/specter-desktop/docker-compose.yml:5]: I've just `docker pull`ed it, which displayed a sha256 that works for all architectures.

**Review by willcosgrove — DISMISSED** [review]

**Comment by nolim1t** [comment]
You can change this to [v1.4.0](https://hub.docker.com/layers/lncm/specter-desktop/v1.4.0/images/sha256-88293f85e5cc86f8616afe5375ad1aa8a52c472129302ff13d6e0c40857681c9?context=explore) instead

**Comment by nolim1t** [comment]
You can also try [v1.4.1](https://hub.docker.com/layers/lncm/specter-desktop/v1.4.1/images/sha256-33a07e609d840e45e910532a248dd07be29d061fd26fedd64894b8fbc9d79bc5?context=explore) as well ;-)

**Comment by AaronDewes** [comment]
Seriously???? I'll stop updating this PR until we actually are ready to release the next version, if the Specter Team starts to make releases every day.

**Comment by nolim1t** [comment]
lol yeah.. anyway they haven't really updated this since the last release

**Review by badokun — APPROVED** [review]
Looks good. Just a version bump on the docker image

**Comment by lukechilds** [comment]
Has anything changed with regards to regtest?

This is tested and working ok for me on mainnet but if you run it on regtest on umbrel-dev Specter doesn't appear to connect to Bitcoin Core.

**Review by lukechilds — APPROVED** [review]

**Comment by nolim1t** [comment]
That could possibly be an issue with specter itself if it's not working in regtest mode.

**Pull request merged**